docs/APIReference-Language.md

```diff
@@ -106,9 +106,9 @@ export class Source {
 ```
 
 A representation of source input to GraphQL. The name is optional,
-but is mostly useful for clients who store GraphQL documents in
+but is most useful for clients who store GraphQL documents in
```

Comment:
suggestion: Either word would be correct, but they convey different meanings ("generally" vs "to the greatest degree").

I think dropping the word altogether would make the sentence clearer and make the use of "but" justifiable. We should also either drop the comma before "but" or add "it" after "but". A comma is only used if joining two independent clauses. I think this would be fine:

> The name is optional, but it is useful...

Reply:
Agreed on omitting 👍 . Also updated